app/templates/components/global-footer/global-footer-bottom-bar.hbs

```diff
@@ -13,8 +13,8 @@
 			ns='design-system'
 		}}}
 	</div>
-	<div class="wds-global-footer__bottom-bar-row wds-has-border-top global-footer-full-site-link-wrapper">
-		<a {{action 'fullSiteClicked' preventDefault=false}} href="?useskin=oasis" rel="nofollow" class="wds-global-footer__button-link">
+	<div class="wds-global-footer__bottom-bar-row wds-has-border-top">
+		<a {{action 'fullSiteClicked' preventDefault=false}} href rel="nofollow" class="wds-global-footer__button-link">
```

Comment:
is empty `href` valid?

Reply:
Let's not use an anchor tag if it's not used as a link. FOr such onclick actions I propose to use a button or at least a div with aria "role=button". For better crawlability. You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role